Kickstarter-iOS/Views/Cells/LiveStreamContainerMoreMenuCancelCell.swift

```diff
@@ -0,0 +1,34 @@
+import Library
+import LiveStream
+import Prelude
+
+internal final class LiveStreamContainerMoreMenuCancelCell: UITableViewCell, ValueCell {
+
+  @IBOutlet private weak var titleLabel: UILabel!
+
+  internal func configureWith(value moreMenuItem: LiveStreamContainerMoreMenuItem) {
+    self.titleLabel.text = localizedString(key: "Cancel", defaultValue: "Cancel")
+
+    self.selectedBackgroundView = self.selectionView
+  }
+
+  internal override func bindStyles() {
+    super.bindStyles()
+
+    _ = self
+      |> UITableViewCell.lens.backgroundColor .~ .hex(0x1B1B1C)
```

Comment:
It's the darker grey background of the more menu cell:

<img width="864" alt="screen shot 2017-03-15 at 12 23 14 pm" src="https://cloud.githubusercontent.com/assets/3735375/23944084/4cac68e6-097a-11e7-835e-1debd69b98de.png">